src/hooks/useFileInput/useFileInput.ts

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 export type UseFileInputProps = {
     onUpdate?: (files: File[]) => void;
     onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    multiple?: boolean;
```

Comment:
The `multiple` prop was not initially added, as the hook gives the basic configuration for the native input. That is, if the consumer needs `multiple`, then you need to add it yourself, otherwise it turns out to be a strange proxy that passes the prop through and returns it in the same form without using it in any way in calculations.